ansible/roles/kubernetes-addons/tasks/dns.yml

```diff
@@ -13,6 +13,7 @@
     url=https://raw.githubusercontent.com/GoogleCloudPlatform/kubernetes/master/cluster/addons/dns/skydns-rc.yaml.in
     dest="{{ local_temp_addon_dir }}/dns/skydns-rc.yaml.j2"
     force=yes
+    validate_certs=False
```

Comment:
pls remove and do an overall scrub on changes that are outside the scope of what's needed to add Contiv support. If their is an issue with cert validation, pls open a GH issue and provide an explanation or a a PR that explains and fixes the issue.